Gruntfile.js

```diff
@@ -154,8 +154,6 @@ module.exports = function (grunt) {
     // Project configuration.
     grunt.initConfig({
 
-        // Metadata required for build.
-        build: grunt.file.readYAML('build/build.yml'),
```

Comment:
I don't know much about the whole Rhino deal.  Was _build/build.yml_ only used for building Less-Rhino?  It looks like it was moved to _build/build.yml.old_ anyway.  If we're not using the file at all, is there any benefit to keeping it in the repo?

Reply:
I think it was just used for that. I ran the usual `grunt` tasks and I got no failures. Nor did I see references anywhere else for it. I'll see if I can delete it though.